### PR TITLE
Feature/data routes

### LINKED ISSUE #3: Feature/IntroPage
Intro text, 3 buttons (person, location, blank), button for editing existing story

### PR DESCRIPTION
**Pull request by vriesm060**

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -4,10 +4,22 @@ var router = express.Router();
...
 
-/* GET home page. */
+// GET home page:
 router.get('/', controller.homePage);
 
-// GET detail page:
-router.get('/detail/:id', controller.detailPage);
+// GET new story page:
+router.get('/new-story', controller.newStoryPage);
+
```
Comment (s44s) [inline routes/index.js]: Add: `router.post('/new-story', controller.newStoryPage);`

Redirect to /create-story

File: routes/index.js [hunk 2]
```diff
@@ -4,10 +4,22 @@ var router = express.Router();
...
+// POST create story page:
+router.post('/create-story', controller.createStoryPage);
```
Comment (s44s) [inline routes/index.js]: Replace to: `router.get('/create-story', controller.createStoryPage);`

File: routes/index.js [hunk 3]
```diff
@@ -4,10 +4,22 @@ var router = express.Router();
...
+
+// POST save story page:
+router.post('/save-story', controller.saveStoryPage);
```
Comment (s44s) [inline routes/index.js]: Add: `router.post('/save-story', controller.createStoryPage);`

File: routes/index.js (hunk 1 above)
Comment (vriesm060) [reply routes/index.js]: No I wont :P

**Pull request merged**